src/status_im/ui/screens/wallet/onboarding/setup/views.cljs

```diff
@@ -1,56 +1,59 @@
-            [status-im.ui.screens.wallet.onboarding.setup.styles :as styles]
             [status-im.ui.components.bottom-buttons.view :as bottom-buttons]
             [status-im.ui.components.button.view :as button]
-            [status-im.utils.utils :as utils]
+            [status-im.ui.components.icons.vector-icons :as vector-icons]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.status-bar.view :as status-bar]
+            [status-im.ui.components.styles :as components.styles]
             [status-im.ui.components.toolbar.actions :as actions]
-            [status-im.ui.components.status-bar.view :as status-bar]))
+            [status-im.ui.screens.wallet.components.views :as wallet.components]
+            [status-im.ui.screens.wallet.onboarding.setup.styles :as styles])
+  (:require-macros [status-im.utils.views :as views]))
 
-(defn signing-word [word]
-  [react/view styles/signing-word
-   [react/text {:style           styles/signing-word-text
+(defn signing-emoji [word first?]
+  [react/view (merge styles/signing-emoji-container
+                     (when-not first?
+                       styles/signing-emoji-container-left-border))
+   [react/text {:style           styles/signing-emoji
                 :font            :roboto-mono
                 :number-of-lines 1}
     word]])
 
-(defn display-confirmation [modal?]
-  (utils/show-question
-   (i18n/label :t/wallet-set-up-confirm-title)
-   (i18n/label :t/wallet-set-up-confirm-description)
-   #(re-frame/dispatch [:accounts.ui/wallet-set-up-confirmed modal?])))
-
 (views/defview onboarding-panel [modal?]
   (views/letsubs [{:keys [signing-phrase]} [:get-current-account]]
-    (let [signing-words  (string/split signing-phrase #" ")
+    (let [signing-emojis (string/split signing-phrase #" ")
           container      (if modal? react/view wallet.components/simple-screen)
           container-opts (if modal? components.styles/flex {:avoid-keyboard? true})]
       [container container-opts
        [wallet.components/toolbar
         {}
         (actions/back-white #(re-frame/dispatch [:wallet-setup-navigate-back]))
-        (i18n/label :t/wallet-set-up-title)]
-       [react/view components.styles/flex
-        [react/view {:style styles/setup-image-container}
-         [react/image {:source (:wallet-setup resources/ui)
-                       :style  styles/setup-image}]]
-        [react/view {:style styles/signing-phrase}
-         (for [word signing-words]
-           ^{:key (str "signing-word-" word)}
-           [signing-word word])]
-        [react/text {:style styles/description}
-         (i18n/label :t/wallet-set-up-signing-phrase)]
+        (i18n/label :t/wallet-setup-title)]
+       [react/view {:style {:flex 1}}
+        [react/view {:style {:flex 1
+                             :flex-direction :column
+                             :align-items :center
+                             :justify-content :center}}
```

Comment:
move to styles?

Reply:
as mentionned in the review notes:
```
I voluntarily didn't moved some of the styling from views to styles namespace because it is totally impractical when working with flexboxes to have to switch inbetween namespaces to understand what is going on.
```